src/request.ts

```diff
@@ -1126,6 +1133,44 @@ class DatastoreRequest {
     );
   }
 
+  private _findLargeProperties(
```

Comment:
Usually some documentation with brief explanation of methods purpose, params and return value(s) should precede the method.

Reply:
added, please provide your suggestion to remove/add anything